fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1077,6 +1078,10 @@ showRoutineLoadTaskStatement
         WHERE expression
     ;
 
+showCreateRoutineLoadStatement
+    : SHOW CREATE ROUTINE LOAD FOR (db=qualifiedName '.')? name=identifier
```

Comment:
Like `show create table xxx` , maybe the `FOR` is not necessary.

Reply:
> Like `show create table xxx` , maybe the `FOR` is not necessary.

@jaogoy What do you think?